scripts/cfgCommons.py

```diff
@@ -74,19 +74,19 @@ def __init__(self, args):
                                     setattr(self, optionUniqueName, newValue)
                             else:
                                 setattr(self, optionUniqueName, newValue)
-            except IOError:
-                logger.error('Cannot load config file %s', args.configFileName)
-                sys.exit(1)
+                except IOError:
+                    logger.error('Cannot load config file %s', common_configFilePath)
+                    sys.exit(1)
 
         # Set command line parameters
         # command line parameters are having precedence, therefore they are set the last
         for arg in vars(args):
             if hasattr(args, arg) and getattr(args, arg): # attribute is present and not empty
                 if hasattr(self, arg):
-                    logger.warning("Overwriting config file parameter '%s' with value '%s' from comman line argumets.", arg, getattr(args, arg))
+                    logger.warning("Overwriting config file parameter '%s' with value '%s' from command line argumets.", arg, getattr(args, arg))
                 setattr(self, arg, getattr(args, arg))
-        if hasattr(self, 'common_output_config'):
-            self.saveConfiguration(getattr(self, 'common_output_config'))
+#        if hasattr(self, 'common_output_config'):
+#            self.saveConfiguration(getattr(self, 'common_output_config'))
```

Comment:
Why?

Reply:
We do not want to save configuration automatically while loading it - do that inside the script instead.